clouddriver-core/src/test/java/com/netflix/spinnaker/clouddriver/security/AccountDefinitionMapperTest.java

```diff
@@ -53,9 +54,15 @@ void canConvertAdditionalAccountTypes() throws JsonProcessingException {
     assertEquals(account, mapper.deserialize(mapper.serialize(account)));
   }
 
+  @Test
+  void canConvertJacksonizedAccountTypes() throws JsonProcessingException {
+    var account = ValueAccount.builder().name("james").value("meowth").build();
+    assertEquals(account, mapper.deserialize(mapper.serialize(account)));
+  }
+
   @Test
   void canDecryptSecretUris() {
-    var data = "{\"type\":\"test\",\"name\":\"bar\",\"password\":\"encrypted:noop!v:hunter2\"}";
```

Comment:
It's important (or at least makes backwards-compatibility/upgrading easier) for "the old way" to still work if at all possible.